src/javascript/EditPanelContainer/SiteData/SiteData.jsx

```diff
@@ -6,10 +6,12 @@ import {connect} from 'react-redux';
 import * as PropTypes from 'prop-types';
 import {SiteInfo} from '@jahia/react-apollo';
 
-export const SiteData = ({notificationContext, t, site, lang, children}) => {
-    return (
-        <SiteInfo siteKey={site} displayLanguage={lang}>
-            {({siteInfo, error, loading}) => {
+export const withSiteInfo = Children => {
+    const SiteData = props => {
+        const {notificationContext, t, site, lang} = props;
```

Comment:
const SiteData = ({notificationContext, t, site, lang}) => {

Reply:
no i need to access to all props in the component.

HOC pattern :p

Reply:
Mmmm, I see some anti PATTERN. For example the HOC T used in SiteData shouldn't be passed to the child, you can use HOC T for the child in question directly.

Anyway...